scripts/ci/pre_commit/pre_commit_update_versions.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
```

Comment:
Thanks for Python and not another Bash script.

Reply:
It;s easier in python  and  I can do both.